subsys/bluetooth/host/audio/Kconfig

```diff
@@ -0,0 +1,108 @@
+# Bluetooth Audio configuration options
+
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_AUDIO
+	bool "Bluetooth Audio support"
+	help
+	  This option enables Bluetooth Audio support. The specific
+	  features that are available may depend on other features
+	  that have been enabled in the stack, such as Periodic
+	  Advertisement for Broadcast and L2CAP Dynamic Channel
+	  for Unicast.
+
+if BT_AUDIO
+
+config BT_ISO
+	# Virtual/hidden option
+	bool
+
+if BT_CONN
+
+config BT_AUDIO_UNICAST
+	bool "Bluetooth Unicast Audio Support"
+	select BT_SMP
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_ISO
+	select BT_GATT_DYNAMIC_DB
+	select BT_GATT_CACHING
+	select BT_EATT
+	help
+	  This option enables support for Bluetooth Unicast Audio using
+	  Isochronous channels.
```

Comment:
I realize that the only (spec) defined use case of ISO is for audio, but I think we should consider decoupling audio and ISO, as ISO may have other use cases than just audio, and it would be somewhat weird having to enable audio in order to use ISO for non-audio use cases.

Reply:
Yep, but for now I decided to leave it under audio to make it simple, if in the future there are other users we can decouple the options and perhaps have a dedicated directiory for iso or just its handling to the directory bellow.